vendor/izinto/izi-io-4840.yaml

```diff
@@ -0,0 +1,107 @@
+name: izi-io 4840
+description: Multi-I/O Edge Computing Controller with LoRaWAN. The top-hat rail mounted device features 8 analog / RTD / digital inputs, 4 digital outputs as well as industry-standard bus interfaces such as CAN and Modbus.
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '1.1'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '1.0'
+    numeric: 1
+    # Corresponding hardware versions (optional)
+    hardwareVersions:
+      - '1.1'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+    profiles:
+      EU863-870:
+        # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: izi-io-4840-profile-eu868
+        #lorawanCertified: true
+        ##codec: izi-io-4840-codec
```

Comment:
Can we help you with the codec?

Reply:
Not at the moment. Since the node is universally programmable (Soft-PLC), there is no predefined message format, therefore no codec. We would release a codec if there's a standard application for a specific use case. In that case we would bundle izi-io + e.g. appropriate external sensors into a separate product "box" and provide the codec here.

Reply:
Okay but in that case you might want to add a firmware version of a specific example application. But you can also do that later, that's fine.